homeassistant/components/wemo/__init__.py

```diff
@@ -44,12 +47,15 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+HostPortTuple = Tuple[str, Optional[str]]
```

Comment:
Is the port really a string?

Reply:
No it is not. Nice catch, thank you! I missed the `cv.port()` call. Fixed.